src/components/button/Button.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+function Button ({message, title, enabled}) {
+return(
+    <button disabled={{enabled}}
+    type="button"
```

Comment:
Kijk, hier geef je de naam 'enabled' aan de property die de waarde symboliseert van het attribuut disabled. Dat zorgt voor de verwarring. Want in App.js moet je dan schrijven enabled={false} als een button aan is en {true} als die uit staat. Als je de property de naam 'disabled' geef dan leest en schrijft het makkelijker in App.js